src/unistd.rs

```diff
@@ -114,8 +114,10 @@ pub fn chdir<P: ?Sized + NixPath>(path: &P) -> Result<()> {
 #[inline]
 pub fn chown<P: ?Sized + NixPath>(path: &P, owner: Option<uid_t>, group: Option<gid_t>) -> Result<()> {
     let res = try!(path.with_nix_path(|cstr| {
-        // We use `0 - 1` to get `-1 : {u,g}id_t` which is specified as the no-op value for chown(3).
```

Comment:
I thought about it; but then, we are not documenting all the other functions properly. So, for consistency I removed it. :)

If you do mind, I'll put it in again.